flink-libraries/flink-ml/pom.xml

```diff
@@ -103,6 +103,9 @@
 		<profile>
 			<id>windows</id>
 			<activation>
+				<property>
+					<name>windows</name>
```

Comment:
This profile is never activated from the command-line and doesn't require the property.